### DISCUSSION TITLE
Temporary disable hot reloading in the browser

### DISCUSSION THREAD

**Discussion by dmitrygorbenko** [webpack/webpack#16556] [Q&A] [answered]
Use case:
I need to keep two tabs of the application, one with old version of code, and the other with newer, to compare how they work.
But the "old version" won't stay, it's gonna be refreshed by Hot Reload module each time I made any changes to source code.

How can I disable temporary hot reloading in the browser's tab ?

I don't want to make any changes to the webpack config, that's too obvious and i would not ask this question here. I'm interested only in one browsers's tab.

Thanks

**Comment by alexander-akait** [comment]
It is impossible, you can send a PR with improvement here - https://github.com/webpack/webpack-dev-server/blob/master/client-src/index.js, firstly we try to get query from your entry, then from configuration, you can add additional check and do not allow HMR/other things if you have `?hot=false` in your tab

**Comment by dmitrygorbenko** [comment, marked as answer]
I have a webpack-dev-server of 3.11.2, and I found in index.js file a string "hotreload=false" which stopped hot reloading on the tab.
I know you have version 4.11.1, maybe the "hot=false" works in the same way.
But, so far, I can see on the console messages
```
[WDS] Live Reloading enabled.
[WDS] App updated. Recompiling...
```
but the actual reloading is not happening.
Win.
Thank you fro helping me.
